deploy/gk-deploy

```diff
@@ -162,13 +162,9 @@ while [[ $# -ge 1 ]]; do
     keypos=1
     while [[ $keypos -lt $keylen ]]; do
       case ${key:${keypos}} in
-        o*|-openshift)
-        CLI="oc"
-        if [ "$key" == "--openshift" ]; then keypos=$keylen; fi
-        ;;
```

Comment:
This removal hunk is (arguably) not a syntax cleanup. :-) What's the reasoning? Should it be a separate commit if at all?